### DISCUSSION TITLE
Licensing the UI Crate under Apache 2.0 Following GPUI's Open Source Transition

### DISCUSSION THREAD

**Discussion by tpisto** [zed-industries/zed#8372] [Feature Requests]
Hello! After GPUI was made open-source with the Apache 2.0 license, is it possible to apply the same license to the **ui** -crate that includes components such as Buttons, Checkboxes, etc.? I believe doing so would greatly benefit individuals beginning to incorporate GPUI into their own projects.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
